modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/portlet/action/ViewPagesMVCRenderCommand.java

```diff
@@ -79,7 +79,9 @@ private String _getPortletId(RenderRequest renderRequest) {
 	@Reference
 	private WikiEngineRenderer _wikiEngineRenderer;
 
-	@Reference
-	private WikiPortletToolbarContributor _wikiPortletToolbarContributor;
+	@Reference(
+		target = "(component.name=com.liferay.wiki.web.internal.portlet.toolbar.item.WikiPortletToolbarContributor)"
+	)
+	private PortletToolbarContributor _wikiPortletToolbarContributor;
```

Comment:
Hey @MrJohn1911 , please apply the same approach [here]( https://github.com/liferay/liferay-portal/blob/master/modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/display/context/WikiPagesManagementToolbarDisplayContext.java#L161). Thx